README.md

```diff
@@ -65,7 +58,225 @@ accessible) and register the service type using the system SmartObject located
 at System > Management > SmartObjects > SmartObjects > JavaScript Service
 Provider and run the Create From URL method.
 
-### License
+## Migration from Ava to Vitest
```

Comment:
move the upgrade guide down to the bottom perhaps keep a link at the top.

Add a [[_TOC_]] at the top